dev_guide/builds.adoc

```diff
@@ -549,189 +615,489 @@ source:
 
 // end::using-a-proxy-for-git-cloning-1[]
 
-[[starting-a-build]]
 
-== Starting a Build
-Manually invoke a build using the following command:
+[#using-private-repositories-for-builds]
```

Comment:
You are absolutely right. This is wrong in the current docs, I just moved it around.
Fixing it here: https://github.com/rhcarvalho/openshift-docs/commit/3958b86c7bc49ab3ed9be639ad89d5909378b13c

Probably not a big deal for existing documentation because the text around braces is not output to HTML, and the default anchor name is the same as the one that would be otherwise defined (if it wasn't for the syntax error).